src/Core/Managed/Shared/Channel/IDeepCloneable.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Microsoft.ApplicationInsights.Channel
```

Comment:
Should this interface be in the `Microsoft.ApplicationInsights` namespace? I realize that it's associated with the work you're doing in channels/sinks, but it seems more general purpose.